Assets/Scripts/Models/Furniture.cs

```diff
@@ -305,6 +317,19 @@ static public Furniture PlaceInstance(Furniture proto, Tile tile)
 
         }
 
+        // Call LUA install scripts
+        if(obj.installActions != null )
```

Comment:
I suggest you either use brackets or indent the line after the if. It makes it more clear

```csharp
if(...)
{
    DoSomething();
}